src/Heading/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import "./styles.css";
+import logo from "../logo/React.webp";
+import usericon from "../logo/usericon.png";
+import searchicon from "../logo/searchicon.png";
+
+const Heading = ({}) => (
+  <div className="header">
+    <div className="headerComponents">
+      <img src={logo} width="50" height="50" />
+    </div>
+    <div className="searchComponentClass">
+      {" "}
```

Comment:
just curious.. why is this needed here?

Reply:
prettier got this when I saved it. something to do with formatting then any functional impact.